### PR TITLE
LRQA-40545 | master

### PR DESCRIPTION
**Pull request by kyle-miho**
https://issues.liferay.com/browse/LRQA-40545

### CODE REVIEW

**Comment by kyle-miho** [comment]
Previously tested here: [#477](https://github.com/anthony-chu/liferay-portal/pull/477).

**Comment by kyle-miho** [comment]
ci:test;sf

File: portal-web/test/functional/com/liferay/portalweb/macros/LayoutPageTemplates.macro [hunk 1]
```diff
@@ -59,6 +59,18 @@
 		</execute>
 	</command>
 
+	<command name="gotoPortletConfiguration">
+		<execute function="MouseOver" locator1="Fragment#PAGE_TEMPLATE_NAVIGATION_PORTLET">
+			<var name="key_fragmentName" value="${fragmentName}" />
+		</execute>
+
+		<execute function="Pause" locator1="2000" />
```
[the remark this answers is not in the document]
Comment (kyle-miho) [reply portal-web/test/functional/com/liferay/portalweb/macros/LayoutPageTemplates.macro]: @anthony-chu yea you are correct, it makes more sense to wait for element than pausing.

**Comment by anthony-chu** [comment]
Thanks @kyle-miho.

Pushed to BChan.
https://github.com/brianchandotcom/liferay-portal/pull/59690

**Pull request closed without merging**